package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "next",
-  "version": "2.0.0-beta.24",
+  "version": "2.0.0-beta.23",
   "description": "Minimalistic framework for server-rendered React applications",
```

Comment:
@rauchg Think minimalistic is not a word 🤔 
http://www.malcolmcoles.co.uk/blog/minimalistic-not-word-minimalist-i/